ChangeLog.d/9618.txt

```diff
@@ -0,0 +1,9 @@
+Removals
+   * Previously deprecated build symbol `MBEDTLS_ENTROPY_HARDWARE_ALT` is
+     removed.
+
+Features
+   * The new symbol `MBEDTLS_PLATFORM_GET_ENTROPY_ALT` is added to allow the
+     defintion of a custom callback function named
+     `mbedtls_platform_get_entropy_alt()` which allows to gather entropy
+     data from some hardware device.
```

Comment:
The hardware device is commonly called a “true hardware random generator” (TRNG). (”True“ is not a good name, but it is commonly used.) This is the term that users will look for, so the changelog entry should mention it.

Suggested wording:

> To supply a platform-specific entropy source, define the compilation option MBEDTLS_PLATFORM_GET_ENTROPY_ALT and provide the callback function mbedtls_platform_get_entropy_alt(). This function should typically access a TRNG ("true hardware random generator") device on bare-metal platforms, or call an operating system function to obtain cryptographic-quality random data.